packaging/arch/PKGBUILD

```diff
@@ -1,30 +1,26 @@
-# $Id$
-# Maintainer: Timothy Redaelli <timothy.redaelli@gmail.com>
+# Maintainer: aimileus <me at aimileus dot nl>
+# Maintainer: Maciej Borzecki <maciek.borzecki@gmail.com>
+# Contributor: Timothy Redaelli <timothy.redaelli@gmail.com>
 # Contributor: Zygmunt Krynicki <me at zygoon dot pl>
-# Contributor: Maciej Borzecki <maciek.borzecki@gmail.com>
 
-pkgbase=snapd
-pkgname=snapd-git
-pkgver=2.31.1
+pkgname=snapd
+pkgdesc="Service and tools for management of snap packages."
+depends=('squashfs-tools' 'libseccomp' 'libsystemd')
+optdepends=('bash-completion: bash completion support')
+pkgver=2.31.1.r619.g3c932a4c5
 pkgrel=1
-arch=('i686' 'x86_64')
```

Comment:
No more i686 builds?

Reply:
Yeah, i686 is no longer supported by Arch as a distro. When building, the arch check can be disabled by `makepkg -A`.